new-frontiers-14/frontier-station-14
# Chemistry Locker Hotfix

## About the PR



Pushes the chemistry locker fill to an NF entity.

## Why / Balance


Filled wall locker variant was inheriting from a filled variant of a non-wall locker, which caused some problems - the wall locker could be dragged, anchored, etc.

## How to test


* Spawn chemistry wall locker.
* Spawn filled chemistry wall locker.
* Try to drag both, neither should drag.
* Spawn filled standing chemistry locker.
* Drag, anchor, open, examine contents.

## Media



## Breaking changes


**Changelog**



No changelog, trivial fix.
